package on2019_12.on2019_12_24_Codeforces_Round__610__Div__2_.C__Petya_and_Exam;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("src/test/java/on2019_12/on2019_12_24_Codeforces_Round__610__Div__2_/C__Petya_and_Exam/C. Petya and Exam.json"))
			Assert.fail();
	}
}
